### DISCUSSION TITLE
Have graphql query support

### DISCUSSION THREAD

**Discussion by felipeospina21** [darrenburns/posting#89] [Ideas]
Would be great to have proper gql query support with syntax highlight, autocomplete and schema documentation. This way we can move away completely from insomnia/postman.

**Comment by adriangalilea** [comment]
My workflow currently uses both HTTP and GraphQL all the time, so my adoption of tools like this is limited by the support of Graphql.

Btw @felipeospina21 I'd use the term `GraphQL` fully, since `GQL` is a language on it's own and may be confusing (in fact there's 2 GQL languages, one from google and the recent Graph Query Language similar to SQL for Graph db's)

**Comment by tusharsadhwani** [comment]
great original thread btw XD

good thing is a graphql client is fairly simple – a basic one will just pass the request body and key values by jsonifying it, and then check the response "message" field to show the body. it's the servers that are painful to make.

**Reply by tusharsadhwani** [reply]
the tricky bit can be introspection - which v1 shouldn't be concerned with IMO.

**Comment by rumbledennis** [comment]
Is this still relevant? I'm currently trying to add support for GraphQL, including schema fetching and autocompletion.

**Reply by felipeospina21** [reply]
it is to me.

**Reply by rumbledennis** [reply]
PR is created https://github.com/darrenburns/posting/pull/372
